Controls/Cafe/Customer1.c

```diff
@@ -0,0 +1,29 @@
+#include "Cafe.h"
+
+/**
+ * Customer 1 wants to buy something!
+ */
+
+/**
+ * @brief Get the FOURTH item and cost on the menu (Cafe.h). Return the name and cost
+ * of the item to the pointers passed into the function. Add the cost to the business 
+ * revenue and print the current business revenue.
+ */
+void Customer1_checkout(char* name, int* cost) {
+	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+}
+
+/**
+ * @brief Acquire the register. Call checkout. Print the name and cost of the item.
```

Comment:
With the current order of things, the updated business revenue will be printed before the item name and cost. That might be a bit confusing, but if you're doing it just to demonstrate returning stuff through arguments, I think it's fine.

Reply:
It's probably fine.